_data/projects/Getting_Started_with_Kotlin.yml

```diff
@@ -0,0 +1,12 @@
+name: Getting_started_with_Kotlin
+desc: Kotlin Started from basics 
+site: https://github.com/PrajjawalBanati/Kotlin_Language
+tags:
+- kotlin
+- kotlin-language
+- kotlin-multiplatform
+- kotlin-android
+- kotlin-functional-programming
+upforgrabs:
+  name: help-wanted
```

Comment:
@PrajjawalBanati  `help wanted`, not help-wanted

Reply:
See it is https://github.com/PrajjawalBanati/Kotlin_Language/labels/help%20wanted